elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -33,16 +32,6 @@
         NONE
     }
 
-    /**
-     * Wrap the opaque user.
-     *
-     * @param opaqueUser the opaque user
-     * @return wrapped user context
-     */
-    default User accessUser(Object opaqueUser) {
-        return new User(opaqueUser);
```

Comment:
@aklish We used this for hydrating the opaqueUser bean in the context of the current Hibernate transaction.
How will lifecycle work then without this?